x-pack/test/functional/services/aiops/dashboard_embeddables.ts

```diff
@@ -33,6 +35,14 @@ export function AiopsDashboardEmbeddablesProvider({ getService }: FtrProviderCon
       });
     },
 
+    async assertChangePointChartInitializerNotExists() {
+      await retry.tryForTime(10 * 1000, async () => {
+        await testSubjects.missingOrFail('aiopsChangePointChartEmbeddableInitializer', {
+          timeout: 1000,
+        });
+      });
+    },
```

Comment:
Let's make this method more flexible and aligned with the rest of our FTR services:
```suggestion
    async assertChangePointChartInitializerExists(expectExist = true) {
      await retry.tryForTime(10 * 1000, async () => {
        if (expectExist) {
          await testSubjects.existOrFail('aiopsChangePointChartEmbeddableInitializer', {
          timeout: 1000,
        }) else {
        await testSubjects.missingOrFail('aiopsChangePointChartEmbeddableInitializer', {
          timeout: 1000,
        });
      });
    },
```

Reply:
Refactored in https://github.com/elastic/kibana/pull/217178/commits/71ac1d8a0b2e3f8e6bb8d364baf4d525bb59b081